### PR TITLE
add migrate, go mod tidy and dependabot.yml

### PR DESCRIPTION
**Pull request by vladimirvshivkov**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,15 +1,17 @@
 module github.com/stackmon/otc-status-dashboard
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.2
```
Comment (sgmv) [inline go.mod:5]: Please use go version 1.22 exactly. Because firstly we need to check, that 1.23 can be possible to use.

Comment (sgmv) [reply go.mod:5]: Just remove this line :)

Comment (sgmv) [inline go.mod:5]:
```suggestion
toolchain go1.22.6
```

Comment (vladimirvshivkov) [reply go.mod]: set toolchain go1.22.9

Comment (vladimirvshivkov) [reply go.mod]: set toolchain go1.22.9

**Review by sgmv — APPROVED** [review]

**Pull request merged**